hello-nextflow/hello-config.nf

```diff
@@ -30,7 +31,7 @@ workflow {
     collectGreetings(convertToUpper.out.collect(), params.batch)
 
     // emit a message about the size of the batch
-    collectGreetings.out.count.view { "There were $it greetings in this batch" }
+    collectGreetings.out.count.view { "There were ${it} greetings in this batch" }
```

Comment:
are the extra curlies necessary here?

Reply:
also should we be explicitly naming the variable like we do later with `line -> line[0]` etc?

Reply:
See my point above about curly braces. As for the implicit `it`, yes I would recommend as a best practice that you declare an explicit name such as `line`. Right now it's only a "paranoid" warning but it will eventually become an error

Reply:
I thought I got all of the `$it`s in https://github.com/nextflow-io/training/pull/538 - must have missed this one.